Components/Configs/Cdts/CdtsEnvironmentList.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace GoC.WebTemplate.Components.Configs.Cdts
+{
+    public class CdtsEnvironmentList
```

Comment:
This class is not used and should be removed.

Reply:
done